src/WorldStorage/NBT.h

```diff
@@ -0,0 +1,82 @@
+
+#pragma once
+
+#include <variant>
+
+#include "NBTDef.h"
+/*
+This class represents a mutable NBT tree. Everything is stored in a variant.
+*/
+
+// fwd:
+class cParsedNBT;
+class cFastNBTWriter;
+
+// fwd:
+class cNBTContent;
+class cNBT;
+namespace NBT
+{
+	using cByteArray = std::vector<char>;  // Todo: Make this a Int8
+	using cIntArray = std::vector<Int32>;
+	using cList = std::vector<cNBT>;
```

Comment:
A list isn't just an array of tags, all the entries need to have the same tag type.

Reply:
cNBT::Push enforces this I believe

Reply:
Actually yes, BUT you can create a cList object and create a cNBT from this and have a list with various types of entries

Reply:
i tried to fix this